### DISCUSSION TITLE
Status of the project

### DISCUSSION THREAD

**Discussion by WhoAteDaCake** [rome/tools#1662] [Q&A] [answered]
Is the project is still active? It's been 28 days since the last commit

**Comment by ematipico** [comment, marked as answer]
> Is the project is still active? It's been 28 days since the last commit

Hi @WhoAteDaCake !
The project is still active, no fears! 😃 

The team is currently doing lot of prototyping and R&D around Rust. As we [decided to shift to Rust](https://discord.com/channels/678763474494423051/788153062211715092/872162728188444703) as main programming language, we are evaluating techniques, architectures, patterns and libraries to use with Rust. 

I don't know when we will start working on a public repository. Although we are making good progresses!

**Reply by WhoAteDaCake** [reply]
Thanks for the update!
